os/net/mac/tsch/tsch-timeslot-timing.c

```diff
@@ -47,7 +47,21 @@
 #include "net/mac/tsch/tsch.h"
 
 /**
- * \brief The default timeslot timing in the standard is a guard time of
+ * \brief TSCH timing attributes and description. All timings are in usec.
+ *
+ * CCAOffset   -> time between the beginning of timeslot and start of CCA
+ * CCA         -> duration of CCA (CCA is NOT ENABLED by default)
+ * TxOffset    -> time between beginning of the timeslot and start of frame TX (end of SFD)
+ * RxOffset    -> beginning of the timeslot to when the receiver shall be listening
+ * RxAckDelay  -> end of frame to when the transmitter shall listen for ACK
+ * TxAckDelay  -> end of frame to the start of ACK tx
+ * RxWait      -> time to wait for start of frame (Guard time)
+ * AckWait     -> min time to wait for start of an ACK frame
+ * RxTx        -> receive-to-transmit switch time (NOT USED)
+ * MaxAck      -> TX time to send a max length ACK
+ * MaxTx       -> TX time to send the max length frame
+ *
+ * The default timeslot timing in the standard is a guard time of
```

Comment:
It'd be nice to have a reference to IEEE802.15.4-2015, something like "The timeslot structure in Figure 6-30 of IEEE 802.15.4-2015 is assumed, although some of the time periods have different names."?

Reply:
I added a note about the figure in the standard, thanks!

Reply:
👍